packages/framework/src/Component/Messenger/ExampleDispatcher.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Messenger;
+
+class ExampleDispatcher extends AbstractMessageDispatcher
+{
+    /**
+     * @param string $value
+     */
+    public function dispatchExampleMessage(string $value): void
+    {
+        if ($this->transportDsn === '') {
```

Comment:
I have created the `NullMessageBus` that does not dispatch the message. When the transportDsn is empty, this message bus is used instead of the default one to prevent errors and code duplication. 

in 45caff6488 and c4124c4efe